dartdoc_test/test/extractor_test.dart

```diff
@@ -47,5 +61,31 @@ final x = 1;
 ```''';
       expect(stripComments(leadingSpacedComment), expectText);
     });
+
+    group('line breaks', () {
+      test('can strip CRLF (windows) line breaks', () {
+        final actualText =
+            '/// This is a windows line break comment.\r\n/// ```dart\r\n/// final x = 1;\r\n/// ```';
+        final expectText = '''
+This is a windows line break comment.
+```dart
+final x = 1;
+```''';
+
+        expect(stripComments(actualText), expectText);
+      });
+
+      test('can strip LF (linux) line breaks', () {
+        final actualText =
+            '/// This is a linux line break comment.\n/// ```dart\n/// final x = 1;\n/// ```';
+        final expectText = '''
+This is a linux line break comment.
+```dart
+final x = 1;
+```''';
```

Comment:
nit: Consider writing it on the same form as the `actualText` that way it becomes super easy to compare what charaters changed.
```suggestion
        final expectText = '''This is a linux line break comment.\n```dart\nfinal x = 1;\n```''';
```

Reply:
I fixed to so. thank you :)